deploy/gke-marketplace-app/server-deployer/schema.yaml

```diff
@@ -89,7 +89,7 @@ properties:
   modelRepositoryPath:
     type: string
     title: Bucket where models are stored. Please make sure the user/service account to create the GKE app has permission to this GCS bucket. Read Triton documentation on configs and formatting details, supporting TensorRT, TensorFlow, Pytorch, Onnx ... etc.
-    default: gs://triton_sample_models/models
+    default: gs://triton_sample_models/23_02
```

Comment:
Does the models change with every release? Also, is the GCS bucket going to be created for every release before we publish on NGC?

CC @mc-nv